package.json

```diff
@@ -21,7 +21,6 @@
     "docs": "npm run docs:clean && npm run docs:gen",
     "docs:gen": "typedoc --mode file ./src",
     "docs:clean": "shx rm -rf docs",
-    "docs:watch": "watch \"npm run docs\" ./src",
```

Comment:
This version of the `watch` library has an insecurity, which made me realize we don't even need this script, so I removed it.